packages/react/src/analytics/integrations/GA4/GA4.ts

```diff
@@ -26,15 +31,7 @@ import {
   INIT_ERROR,
   MESSAGE_PREFIX,
   NON_INTERACTION_FLAG,
-  OPTION_DATA_LAYER_NAME,
```

Comment:
Why were these options removed? The idea was to use them when accessing the `options` variable to avoid hard-coding the property names.